lib/simplifyExpression/basicsSearch/removeAdditionOfZero.js

```diff
@@ -1,30 +1,32 @@
-const clone = require('../../util/clone');
+const {canApplyRule, applyRule} = require('math-rules');
```

Comment:
It doesn't have to be tedious.  We can use codemods to do it automatically for us.  I'd like to minimize the number of changes in this branch so that it's easier to review and easier to merge back into `master.